### DISCUSSION TITLE
Release tagging

### DISCUSSION THREAD

**Discussion by DeltaCopy** [Bali10050/Darkly#7] [Ideas]
Are there plans to start publishing releases ?

I've setup a new build on the Fedora Copr.

So far this builds and installs successfully on Fedora 40 and users can easily enable the Copr and install via dnf.
Builds on both Fedora 41 and Rawhide are failing currently due to mirror/checksum issues, so will wait until that is resolved for now.

In order for this to work a release is required to be published.

This will also help with building from the AUR if that is also in the pipeline.

**Comment by Bali10050** [comment]
<https://github.com/Bali10050/Lightly/releases/tag/v0.5.0>

Is this good for the purpose?

**Reply by DeltaCopy** [reply]
Yes.

I started a new build using the newly published release tag and all succeeded on Fedora 40, 41 and Rawhide.

Thanks.
